### ISSUE TITLE
add script to compress icons

### ISSUE THREAD

**Issue by Beatso** [LittleImprovementsCustom/LittleImprovementsCustom#140]
to use node.js tinypng api: https://tinypng.com/developers/reference/nodejs

would enable team members could compress icons by running an npm script in the command line
more info about npm scripts: https://www.twilio.com/blog/npm-scripts

parsing arguments (which icon to compress): https://nodejs.org/en/knowledge/command-line/how-to-parse-command-line-arguments/

> This issue was created by an automation. It was authored in Discord by Beatso#1815, in Beatso's Resource Packs #beatso-spam.

**Comment by Chromus-dev** [comment]
We should have a similar feature on ve

**Comment by Beatso** [comment]
yeah. i'm thinking of creating a CLI or something, in which case the user can just install that. if not, i'll add the same script to VE
